OrbitCore/TracepointInfoManager.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
We are trying to get rid of `OrbitCore` and to at least not add anything to it. In what module is this used? If only `OrbitGl` this can be in `OrbitGl`.

Reply:
Yes, it is used only in OrbitGl. I will move it. Thanks!